kratos/python_scripts/json_output_process.py

```diff
@@ -297,12 +297,12 @@ def ExecuteFinalizeSolutionStep(self):
                                             data["RESULTANT"][variable_name ][-1] += aux
                             elif variable_type == "Vector":
                                 if not self.resultant_solution:
-                                    data[node_identifier][variable_name].append(value)
+                                    data[node_identifier][variable_name].append(self.__kratos_vector_to__python_list(value))
```

Comment:
Is the double underscore between `to` and `python` intentional?
```suggestion
                                    data[node_identifier][variable_name].append(self.__kratos_vector_to_python_list(value))
```

Reply:
Good catch. I'm not the original author of this code, but I would guess it wasn't intentional. Even though it's not directly related to the actual bug fix, I think it's a good idea to remove the excess underscore now that we've spotted it. Done.